### DISCUSSION TITLE
增加管理服务器Plugins

### DISCUSSION THREAD

**Discussion by Leonmmcoset** [MCSLTeam/MCSL2#144] [功能建议]
我想在MCSL 2中就能查看我的世界服务器Plugins（例如Auth Me等插件）的信息

---

### 再大胆想

调用Spigot等Plugins网站（类似如PCL 2）可以实现下载或更新Plugins。

**Comment by AresConnor** [comment]
本来是有模组/插件广场的插件的，但是因为开发进度丢失了，导致项目搁置了，后期有机会再捡起来把（

**Reply by AresConnor** [reply]
https://github.com/MCSLTeam/ModPlaza_Plugin
